README.md

```diff
@@ -13,10 +15,7 @@ We love contributions that help us, so feel free to! You won't be rewarded for t
 See the [Contributing](./CONTRIBUTING.md) page for more information.
 
 ## Where are your docs?
-SoonTM
-
-## Why doesn't it work?
-We plan on launching a 1.0 version which will work, until that comes out you just have to sit and look at the code and predict what functions are gonna come out next.
+Have functions and attributes you not sure about? Check our docs [here](https://epikcord-guide.vercel.app/)
```

Comment:
By this I meant that the PyPi version being old and broken and throwing old errors, not all classes being here etc.